File: 485.MaxConsecutiveOnes.cpp
class Solution
{
public:
    int findMaxConsecutiveOnes(vector<int> &nums)
    {
        int count = 0;
        int max_count = 0;
        int n = nums.size();
        for (int i = 0; i < n; i++)
        {
            if (nums[i] == 1)
            {
                count++;
                max_count = max(count, max_count);
            }
            else
            {
                count = 0;
            }
        }
        return max_count;
    }
};

File: 485.MaxConsecutiveOnes.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        max_count = 0
        n = len(nums)
        for i in range(n):
            if nums[i]==1:
                count+=1
                max_count = max(count,max_count)
            else:
                count = 0
        return max_count